examples/pairwise_roi_alignment.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding: utf-8 -*-
+"""Functional alignment on a pair of subject
+===================================================================
+
+In this tutorial, we show how to better predict new contrasts for a target /
+    subject using source subject corresponding contrasts and data in common.
+
+We mostly rely on python common packages and on nilearn to /
+    handle functional data in a clean fashion.
```

Comment:
State that this is bound to one region, to outline the difference with the previous example.